packages/ghc/build.sh

```
@@ -0,0 +1,20 @@
+
+# cd into extracted source directory
+cd ghc-%{version}
+
+# Remove nofib benchmark suite (contains hardlink entries that can cause staging issues)
+rm -rf nofib
+
+# Configure GHC (configure script is included in the source tarball)
+./configure --prefix=/usr
+
+# Build using hadrian (default for GHC 9.x)
+./hadrian/build -j$(nproc) --flavour=quickest --disable-split-objs --enable-shared
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)